### ISSUE TITLE
batch_size must be 1?

### ISSUE THREAD

**Issue by Gakkkkkki** [milesial/Pytorch-UNet#259]
I changed the batch_size(1 -> 32) and learning rate(0.0001 -> 0.0032), but code didn't work.

**Comment by Gakkkkkki** [comment]
I have solved this problem. I used images with different sizes.
